src/main/java/edu/wpi/first/pathweaver/CreateProjectController.java

```diff
@@ -0,0 +1,77 @@
+package edu.wpi.first.pathweaver;
+
+import java.io.File;
+import java.util.List;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.TextField;
+import javafx.scene.layout.VBox;
+import javafx.stage.DirectoryChooser;
+
+public class CreateProjectController {
+
+  @FXML
+  private VBox vBox;
+  @FXML
+  private TextField directory;
+  @FXML
+  private Button browse;
+  @FXML
+  private TextField timeStep;
+  @FXML
+  private TextField maxVelocity;
+  @FXML
+  private TextField maxAcceleration;
+  @FXML
+  private TextField maxJerk;
+  @FXML
+  private TextField wheelBase;
+  @FXML
+  private Button create;
+
+  private List<TextField> numericFields;
+
+  @FXML
+  private void initialize() {
+    numericFields = List.of(timeStep, maxVelocity, maxAcceleration, maxJerk, wheelBase);
+    // TODO: validate numbers only
+    browse.setOnAction(event -> {
+      browseDirectory();
+    });
+    create.setOnAction(event -> {
+      createProject();
+    });
+  }
+
+  private void createProject() {
+    // TODO: prompt user if fields are empty
```

Comment:
Instead of this, I would disable the "Create Project" button entirely until all properties are set and valid

Reply:
That's a good idea, I like that.